metadata.yaml

```
@@ -41,4 +41,4 @@ resources:
     type: oci-image
     description: OCI image for mongodb
     # TODO: Update sha whenever upstream rock changes
-    upstream-source: ghcr.io/canonical/charmed-mongodb@sha256:b4b3edb805b20de471da57802643bfadbf979f112d738bc540ab148d145ddcfe
+    upstream-source: ghcr.io/canonical/charmed-mongodb@sha256:c24916ddf111da240dd3d5508ba8d42f611e9167bb9be67a3acc392a25883047
```

Perhaps the new one:
```suggestion
    upstream-source: ghcr.io/canonical/charmed-mongodb@sha256:7ddb80a3b5ddffa95704a8980fc11037ba1a23273a9805214bc42be9f507107f
```